aiosmtpd/controller.py

```diff
@@ -8,13 +8,16 @@
 
 @public
 class Controller:
-    def __init__(self, handler, loop=None, hostname='::0', port=8025):
+    def __init__(self, handler, loop=None, hostname='::0', port=8025,
+                 thread_timeout=5):
```

Comment:
I think it's good that there is a way to set the timeout value without using environment variables, especially if someone wants to make it dynamic in their codebase. `ready_timeout` is fine with me!